browser/ephemeral_storage/ephemeral_storage_forget_by_default_browsertest.cc

```diff
@@ -4,12 +4,12 @@
  * You can obtain one at https://mozilla.org/MPL/2.0/. */
 
 #include "brave/browser/ephemeral_storage/ephemeral_storage_browsertest.h"
-
```

Comment:
please run `npm run presubmit --fix` locally before pushing things. I think the format checks won't like the removal of this line.